ocaml/file_formats/cmx_format.mli

```diff
@@ -19,11 +19,6 @@
 
 (* Format of .cmx, .cmxa and .cmxs files *)
 
-type import_info_cmi =
```

Comment:
(Maybe we should keep the type
synonyms, if we plan on adding
the phantom type. There are
other occurrences, for the various
file formats.)

Reply:
I'm not sure they would be necessary, I'd aim for the phantom parameter to say the same thing (and not be too long).

Reply:
I concur, keeping the type synonyms
was actually to make the transition
path easier to travel. Without them,
one will likely have to consult the
history.